pages/common/ionic.md

```diff
@@ -0,0 +1,31 @@
+# ionic
+
+> Complete open source SDK to build top quality native and progressive web apps using Angular JS and Apache Cordova.  
+
+- Installation.
```

Comment:
To clarify, it's not about need (beginners would certainly welcome help with installation), but about scope: we have very limited space due to our commitment to provide short pages, and we focus on usage documentation; installation instructions (which often can be quite exhaustive to support multiple platforms, package managers, etc.) can clash with these goals, so we choose not to include them for any of the tools we cover. I hope you can understand :)